pkg/querier/engine/config.go

```diff
@@ -44,10 +46,12 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&cfg.MaxSamples, "querier.max-samples", 50e6, sharedWithQueryFrontend("Maximum number of samples a single query can load into memory."))
 	f.DurationVar(&cfg.DefaultEvaluationInterval, "querier.default-evaluation-interval", time.Minute, sharedWithQueryFrontend("The default evaluation interval or step size for subqueries."))
 	f.DurationVar(&cfg.LookbackDelta, "querier.lookback-delta", 5*time.Minute, sharedWithQueryFrontend("Time since the last sample after which a time series is considered stale and ignored by expression evaluations."))
+	f.BoolVar(&cfg.PromQLExperimentalFunctionsEnabled, "querier.promql-experimental-functions-enabled", false, sharedWithQueryFrontend("True to enable support for experimental PromQL functions."))
```

Comment:
[nit] It can also be set without passing `=true` value, simplify wording: 

```suggestion
	f.BoolVar(&cfg.PromQLExperimentalFunctionsEnabled, "querier.promql-experimental-functions-enabled", false, sharedWithQueryFrontend("Enable experimental PromQL functions."))
```

Shall we include list of experimental functions? How does user know when to use this?

Reply:
Simplified CLI description in https://github.com/grafana/mimir/pull/7057/commits/35db456500dc4d71919e26281d36826b706c2a17.

Reply:
> Shall we include list of experimental functions? How does user know when to use this?

Yes, it's in `about-versioning.md` and I've just added to CHANGELOG too.